c2cgeoportal/scaffolds/create/print/print-apps/+package+/config.yaml.mako

```diff
@@ -3,25 +3,30 @@ throwErrorOnExtraParameters: true
 templates:
     A4 portrait: !template
         reportTemplate: A4_Portrait.jrxml
-        tableData: jrDataSource
         attributes:
-            title: !string {}
-            comments: !string {}
+            title: !string
+                default: ""
```

Comment:
Setting these kind of `defaults` values makes Tomcat raise an error...:

Are you sure that this is correct ?